### PR TITLE
fix: improving main page header

### PR DESCRIPTION
**Pull request by Dbishop15**
## Describe your changes
Improving the main page header.
- Changing the width of the main page header to the full width of the page.
- Fixing the login/signin underline not to be overlapping by changing the text size.
- Fixing the header image to span the full width of the page.

## Issue ticket number and link
SLOP-22
https://tripleten-apiary.atlassian.net/jira/software/projects/SLOP/boards/2?selectedIssue=SLOP-22

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: src/components/header/header.jsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ Header.Profile = () => {
         <div className="flex flex-row h-[24px] gap-2.5 md:gap-1 md:pl-4 md:flex lg:flex sm:hidden xs:hidden">
           <Button
             className={"border-b-2"}
-            size="sm"
+            size="base"
```
Comment (SamPage4427) [inline src/components/header/header.jsx:213]: Login and Signup looks good, however the text should look like the other header links but other than that the underline is correct

File: src/components/header/header.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { Button, LoginModal, SignupModal } from "../index";
 
 export const Header = ({ children }) => {
   return (
-    <header className="flex flex-row w-full relative z-10 pl-5 pr-5 max-w-[1440px] mx-auto justify-between items-center text-lg font-arialRegular x-0 content-center h-20 bg-black text-stone-50 xs:pl-2 xs:pr-2 md:text-md md:pr-2 md:pl-2">
+    <header className="flex flex-row w-full relative z-10 pl-5 pr-5 mx-auto justify-between items-center text-lg font-arialRegular x-0 content-center h-20 bg-black text-stone-50 xs:pl-2 xs:pr-2 md:text-md md:pr-2 md:pl-2">
```
Comment (SamPage4427) [inline src/components/header/header.jsx:20]: Header looks great on all sizes now and spans the full page above 1440px.

**Review by SamPage4427 — COMMENTED** [review]
Everything looks great. Just the one issue with the login and signup and the ticket will be complete.

**Comment by SamPage4427** [comment]
image
Login and signup are now being crossed out by the underline

**Review by SamPage4427 — APPROVED** [review]
Everything looks great. The ticket is complete. Josh will review it and if he doesn't have any complaints then the request will be pulled into main

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**